src/components/join/Info.jsx

```diff
@@ -1,7 +1,20 @@
+import InfoCard from "./InfoCard";
+
 const Info = () => {
   return (
-    <div className="">
-      <p>Info</p>
+    <div className="w-full l-100 flex">
+      <div className="flex flex-col ml-28 mt-24">
```

Comment:
you can use width instead of using margining and center the whole divider tag on line 5. you can use justify to space the info cards and text.